presentation/src/main/java/com/wednesday/template/presentation/weather/search/SearchFragmentScreenState.kt

```diff
@@ -0,0 +1,13 @@
+package com.wednesday.template.presentation.weather.search
+
+import com.wednesday.template.presentation.base.UIList
+import com.wednesday.template.presentation.base.UIToolbar
+import com.wednesday.template.presentation.screen.MainScreenState
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class SearchFragmentScreenState(
```

Comment:
Let's name this `SearchScreenState`. Having fragment and screen together seems redundant.